app/manifest.go

```diff
@@ -117,6 +119,13 @@ type ManifestKindVersion struct {
 	CustomRoutes map[string]spec3.PathProps `json:"customRoutes,omitempty" yaml:"customRoutes,omitempty"`
 }
 
+type ManifestCustomRoute struct {
+	Group      string                     `json:"group" yaml:"group"`
```

Comment:
I think we should omit this, as `Group` should always match the manifest's `Group` unless I'm missing something.